bin/hera/src/hera.rs

```diff
@@ -24,13 +25,13 @@ impl HeraCli {
     pub fn run(self) -> Result<()> {
         match self.subcmd {
             HeraSubCmd::ExEx(cli) => cli.run(|builder, args| async move {
```

Comment:
it works for now, but generally it means that we will not be able to have more exexes (if there's any more planned for L1) in the same binary, because this binary has a name `hera`.

I think a better approach would be to have `bin/reth` and `bin/hera` binaries, and in `bin/reth` have a CLI argument `--hera` that will enable the ExEx. In `bin/hera`, it's just a standalone Hera binary. Both ExEx and standalone binary should re-use same code.

Reply:
just thinking that if we're going towards the future with L1 node, derivation pipeline, L2 client, proposer, and challenger in one binary, then it would be better to split the components from the beginning

Reply:
let's make an issue for it

Reply:
I see what you're saying here, but couldn't `bin/reth` just import the standalone binary runner defined in `bin/hera` and run it in a separate thread if the `--hera` flag is supplied?

Then we could have the best of both worlds where the `hera` binary allows you to run as standalone or as an exex, and a separate `reth` binary allows you to pass in `--hera`?

Reply:
If in the future there will be more exexes, it would make more sense to spawn them from `reth` binary imo.
For the `hera` binary I don't have a strong opinion on if it should also be able to start as an exex or not.

Reply:
Happy to go with splitting them - for the sake of unblocking, let's do it :)